data/json/mapgen/map_extras/wilderness.json

```
@@ -769,5 +769,44 @@
         { "monster": "GROUP_SWAMP", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 5, "density": 0.5 }
       ]
     }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_worm_sign",
+    "object": {
+      "rows": [
+        "        .........       ",
+        "     ...............    ",
+        "   ...................  ",
+        "  ....................  ",
+        "  ..................... ",
+        " ...................... ",
+        " ...................... ",
+        "........................",
+        "........................",
+        "........................",
+        "...........x............",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        "........................",
+        " ...................... ",
+        " ...................... ",
+        " ...................... ",
+        "  ....................  ",
+        "  ....................  ",
+        "    ................    ",
+        "       ..........       "
+      ],
+      "flags": [ "ALLOW_TERRAIN_UNDER_OTHER_DATA" ],
+      "terrain": { ".": [ [ "t_region_groundcover", 20 ], "t_dirtmound" ], "x": [ [ "t_region_groundcover", 20 ], "t_dirtmound" ] },
```

```suggestion
      "nested": { ".": { "chunks":  [ [ "null", 20 ], "1x1_upturn_earth" ], "x": [ [ "null", 20 ], "1x1_upturn_earth" ] },
```
If you wanted to be needlessly🌟*fancy*🌟 you could use the 1x1 nest below to avoid overwriting the existing terrain
```
  {
    "type": "mapgen",
    "method": "json",
    "nested_mapgen_id": "1x1_upturn_earth",
    "object": { "mapgensize": [ 1, 1 ], "place_ter_furn_transforms": [ { "transform": "upturn_earth", "x": 0, "y": 0 } ] }
  },
  {
    "type": "ter_furn_transform",
    "id": "upturn_earth",
    "terrain": [ { "result": "t_dirtmound", "valid_flags": [ "DIGGABLE" ] } ]
  }
```